.github/workflows/tests.yml

```diff
@@ -72,7 +91,7 @@ jobs:
         run: pip install -e .
 
       - name: Run tests
-        run: pytest tardis ${{ env.PYTEST_FLAGS }}
+        run: pytest tardis -x ${{ env.PYTEST_FLAGS }}
```

Comment:
I don't think we should exit instantly on the first error or failed test. What do other people think?

Reply:
Agreed and Atharva has solved that